### PR TITLE
[AC-479] Format code and fix Lint warnings

### PR DESCRIPTION
**Pull request by f4ww4z**
Issue link: https://issues.openmrs.org/browse/AC-479

Description
1. Formatted the whole project using [OpenMRSFormatter.xml](https://github.com/openmrs/openmrs-core/blob/master/tools/src/main/resources/eclipse/OpenMRSFormatter.xml)
2. Fixed most Android lint warnings
3. Optimized imports

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/models/Location.java [hunk 1]
```diff
@@ -38,14 +39,23 @@
...
-    public Location() {}
+    public Location() {
+    }
 
     public Location(String display) {
         this.display = display;
     }
 
-    public Location(Long id, String name, String parentLocationUuid, String description, String address2, String address1, String cityVillage, String stateProvince, String country, String postalCode) {
+    public Location(Long id,
```
[the remark this answers is not in the document]
Comment (dkayiwa) [reply openmrs-client/src/main/java/org/openmrs/mobile/models/Location.java]: @f4ww4z did you see the above comment?

[the remark this answers is not in the document]
Comment (f4ww4z) [reply openmrs-client/src/main/java/org/openmrs/mobile/models/Location.java]: @dkayiwa alright, I'll fix this soon.

[the remark this answers is not in the document]
Comment (f4ww4z) [reply openmrs-client/src/main/java/org/openmrs/mobile/models/Location.java]: @dkayiwa , I've fixed this issue.

**Comment by dkayiwa** [comment]
@f4ww4z it will be easier to review if you create a separate pull request for each of these categories.

**Comment by f4ww4z** [comment]
@dkayiwa Got it. I will update the issue page with the list of categories, and then do a PR for each of them.

**Comment by dkayiwa** [comment]
@f4ww4z did you see the merge conflicts on this?

**Comment by f4ww4z** [comment]
@dkayiwa Yes I have. I'll wait until other active PRs are resolved, and then Ill work on this PR, and encourage future PRs to format the code properly.

**Comment by f4ww4z** [comment]
Closing temporarily.

**Pull request closed without merging**